src/main/java/adris/altoclef/util/CachedProjectile.java

```diff
@@ -1,29 +1,35 @@
 package adris.altoclef.util;
 
+
 import adris.altoclef.util.csharpisbetter.Timer;
 import net.minecraft.util.math.Vec3d;
 
 import java.lang.reflect.Type;
 
+
+/**
+ * What is this?
```

Comment:
It keeps track of a projectile and its trajectory. This is done to prevent recalculation, and is mostly used in [GoalDodgeProjectiles](https://github.com/adrisj7-AltoClef/altoclef/blob/4973fc01c9a0f28989b6d9da685a8d11d543b990/src/main/java/adris/altoclef/util/baritone/GoalDodgeProjectiles.java#L15).